docs/kyma/01-01-in-a-nutshell.md

```diff
@@ -3,12 +3,12 @@ title: In a nutshell
 type: Overview
 ---
 
-Kyma allows you to extend applications with microservices and Functions. First, connect your application to a Kubernetes cluster and expose the application's API or events securely. Then, implement the business logic you require by creating microservices or Functions, and triggering them to react to particular events or calls to your application's API. To limit the time spent on coding, use the in-built cloud services from the Service Catalog, exposed by open service brokers from such cloud providers as GCP, Azure, and AWS.
+Kyma allows you to extend applications with microservices and Functions. First, connect your application to a Kubernetes cluster and expose the application's API or events securely. Then, implement the business logic you require by creating microservices or Functions, and triggering them to react to particular events or calls to your application's API. To limit the time spent on coding, use the built-in cloud services from the Service Catalog, exposed by open service brokers from such cloud providers as GCP, Azure, and AWS.
```

Comment:
See my [comment above](https://github.com/kyma-project/kyma/pull/9064/files#r456428631) on the length of the sentence.

Reply:
But doe sit mean you would like me to shorten these ones as well?